Author: Morten Thunberg Svendsen <Morten.ThunbergSvendsen@prevas.dk>
Date:   Tue Apr 15 15:15:15 2015 +0200

Vivante driver viv_5.0.11.p4.1 on X11 version fails to match visual ID. Enable
this define in qtbase for more debug info.

Index: qtbase-opensource-src-5.3.1/src/platformsupport/eglconvenience/qxlibeglintegration.cpp
===================================================================
--- qtbase-opensource-src-5.3.1.orig/src/platformsupport/eglconvenience/qxlibeglintegration.cpp
+++ qtbase-opensource-src-5.3.1/src/platformsupport/eglconvenience/qxlibeglintegration.cpp
@@ -40,7 +40,7 @@
 ****************************************************************************/
 
 #include "qxlibeglintegration_p.h"
-
+#define QT_DEBUG_X11_VISUAL_SELECTION
 VisualID QXlibEglIntegration::getCompatibleVisualId(Display *display, EGLDisplay eglDisplay, EGLConfig config)
 {
     VisualID    visualId = 0;
